packages/react-dom/src/test-utils/FizzTestUtils.js

```diff
@@ -120,4 +140,59 @@ function mergeOptions(options: Object, defaultOptions: Object): Object {
   };
 }
 
-export {replaceScriptsAndMove, mergeOptions};
+function stripExternalRuntimeInNodes(
+  nodes: HTMLElement[] | HTMLCollection<HTMLElement>,
+  externalRuntimeSrc: string | null,
+): HTMLElement[] {
+  if (!Array.isArray(nodes)) {
+    nodes = Array.from(nodes);
+  }
+  if (externalRuntimeSrc == null) {
+    return nodes;
+  }
+  return nodes.filter(
+    n =>
+      (n.tagName !== 'SCRIPT' && n.tagName !== 'script') ||
```

Comment:
Why the lower case? SVG?

https://github.com/facebook/react/pull/25437#discussion_r990524693

Reply:
@sebmarkbage 
I'm not sure why this is needed, but I see that `tagName` is sometimes lowercase. I saw some [existing code like this](https://github.com/facebook/react/blob/fa11bd6ecca57d8ddbc728df881f0e165225c04f/packages/react-dom/src/__tests__/ReactDOMFizzServer-test.js#L192-L193), and didn't think too much of it.

Can look into it if it seems suggestive of a larger issue!